content/recipes/findability/updateChemicalIdentities.md

```diff
@@ -0,0 +1,73 @@
+# Update Chemical Identities
+
+ ````{panels_fairplus}
+:identifier_text: FCB053 
+:identifier_link: 'https://w3id.org/faircookbook/FCB053'
+:difficulty_level: 2
+:recipe_type: hands_on
+:reading_time_minutes: 10
+:intended_audience: bioinformatician, data_scientist, data_engineer
+:has_executable_code: yes
+:recipe_name: How to update chemical identities
+```` 
+
+## Main Objectives
+
+The primary purpose of this recipe is:
+
+> Determine chemical identifiers, given a compound
+> Updating chemical compounds in Wikidata
+
+
+## 1 Requirements
+
+1) Bash Knowledge
+2) Groovy Knowledge
+3) WikiStatement Knowledge (Optional)
+
+---
+
+
+## 2 Main Content
+
+
+
+### 2.1 Generating Compound Identifiers
+Updating chemical identifiers requires the existing knowledge of at least one chemical identifier. 
+
+From this identifier, other compound identifiers can be determined using the following script:
+
+```
+groovy -identifier <Identifier> -identifierType <IdentifierType>
+```
+This script is available from the chemistry development kit/
+
+
+### 2.2 Updating Wikidata
+With the available identifiers of a compound the compound can be updated within Wikidata using any of the following:
+
+- Direct edits to the compound page from the compound's Wikidata page using the 'edit' functionality 
+- Quick statements at https://quickstatements.toolforge.org/
+
+
+
+## Conclusion
+This recipe discussed several aspects of updating chemical compounds
+- How to retrieve chemical identifiers given an original identifier
+- How to update chemical identification properties in Wikidata 
+
+
+## Authors
+
+
+| Name                                                                                                                                                                                                                                       | Orcid                                                                                                                        | Affiliation                           | Type                                                                              |                                                              Elixir Node                                                              | Credit Role
```

Comment:
See this comment: https://github.com/egonw/the-fair-cookbook/pull/9#discussion_r683632776